.gitignore

```diff
@@ -10,3 +10,6 @@ dist/*
 yarn.lock
 package-lock.json
 npm-shrinkwrap.json
+
+# IDE related
+.idea
```

Comment:
i'll rebase this out; your personal IDE's garbage files should be globally gitignored, instead of adding it to every project you touch's gitignore